Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -742,7 +759,7 @@ Task<bool> SwitchContentAsync(Page page, bool animated, bool removed = false, bo
 			Current?.SendDisappearing();
 			Current = page;
 
-			((Platform)Element.Platform).NavAnimationInProgress = true;
+			Platform.NavAnimationInProgress = true;
```

Comment:
Seems like this could potentially be null, albeit rarely. Should we be safe?

Reply:
Added null check.